MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+#### These are the current Htsjdk-Next-Beta project maintainers (sorted alphabetically by handle)
+
+-----
+
+* Jay Carey	**@jacarey** 	_jcarey@broadinstitute.org_
+*	Louis Bergelson	 **@lbergelson**	_louisb@broadinstitute.org_
```

Comment:
Maybe a markdown table instead? The spacing looks funny, and with a table even in plain-text could be perfectly visualized...